packages/design-system/src/components/ChoiceList/ChoiceList.jsx

```diff
@@ -7,6 +7,15 @@ import classNames from 'classnames';
 export class ChoiceList extends React.PureComponent {
   constructor(props) {
     super(props);
+
+    if (process.env.NODE_ENV !== 'production') {
+      if (props.multiple) {
```

Comment:
Could you also add a console warning to handle the case for `this.props.choices.length === 1`? The message should be something like: "Use type=checkbox for <ChoiceList> components with only one choice. A single radio button is disallowed because it prevents users from deselecting the field."

Reply:
Thanks, that is a good point. Will add that.